src/Bicep.LangServer/Deploy/DeploymentCollectionProvider.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.ResourceManager;
+using Azure.ResourceManager.Resources;
+using Bicep.Core;
+
+namespace Bicep.LanguageServer.Deploy
+{
+    public class DeploymentCollectionProvider : IDeploymentCollectionProvider
+    {
+        public DeploymentCollection? GetDeploymentCollection(ArmClient armClient, ResourceIdentifier resourceIdentifier, string scope)
+        {
+            if (scope == LanguageConstants.TargetScopeTypeResourceGroup)
```

Comment:
Can replace the method body with a switch expression.

Reply:
Sure, updated as part of this commit:
https://github.com/Azure/bicep/commit/32bed56af070ae3b2c2b494c956f07cf68874667

Thanks!